### ISSUE TITLE
Siam Starting Bias Still 'Not_Forest'?

### ISSUE THREAD

**Issue by cdanger341** [LoneGazebo/Community-Patch-DLL#4031]
_1. Mod version (i.e Date - 4/23):_
2-27

_2. Mod list (if using Vox Populi only, leave blank):_
N/A

_3. Error description:_ 

I have asked around the forum and I still cannot get a straight answer on this. In vanilla Civ 5 Siam's starting bias is set to "Not_Forest" for some reason, and yet in VP their UB (Wat) gives them bonus Culture for forests and jungle tiles near their cities. Also, Siam is in a part of the world known for its lush jungles, so it doesn't make much sense that their starting bias would be "not close to a forest". Instead is should probably be "Jungle/Forest".

However, I know that some Civs in VP have had their starting bias modified from vanilla (Ex: Russia and Sweden), but has Siam ever had this changed? This may be something worth looking into if it has not.

**Comment by cdanger341** [comment]
I looked into the vanilla and modded files and found that Siam does indeed still have the starting bias of "Avoid Forests" in VP. I added the line in the VanillaLeaders xml file to change it to "Region_Jungle" and started several games on Pangaea, continents, and Communitas to see if it works, and it sure did! Siam was actually starting in jungle/forested areas rather than plains/deserts/tundras/etc. that I usually see them start in. Is it intended that Siam keep their current starting bias given their UB and geographical history?

**Comment by Phalanx1** [comment]
Weird... I suggested start bias changes (#3343) a while ago, and Gazebo implemented them soon after. For some reason the changes got rolled back (or maybe he only implemented some of the them?), I guess.

**Comment by cdanger341** [comment]
Yes, I saw your suggestions and wondered why they were not implemented. I went and implemented them myself and on large/huge maps it seems to make certain civs perform better when they start on parts of the map more suited for their Uniques.
